keps/sig-auth/740-service-account-external-signing/README.md

```diff
@@ -280,10 +275,6 @@ to implement this enhancement.
 ##### Integration tests
```

Comment:
As part of your alpha implementation you committed to add integration tests, I'm missing explicit links [see template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#integration-tests) which point to those added tests.